filtered_topic_name: /skidpad_detection
inverse_flag: 0           # 1: inverse matrix; 0: DO NOT inverse. inverse means car need turn right a little 
input_topic: /velodyne_points
no_ground_point_topic: /points_no_ground
ground_point_topic: /points_ground
all_points_topic: /all_points

clip_height: 2.0          # ?
sensor_height: 0.12       # 0.18   0.135        这里存在的意义是把低于这个-sensor_height的点都先剔除掉  0.12
min_distance: 0.1         # unit?
max_distance: 100.0       # default: 100.0      seems no use

num_iter: 10              # iteration num 3  5   拟合平面次数   越高越精准 同时消耗越高
num_lpr: 5                # seeds num           地面种子数选取5    越大精度越高
th_seeds: 0.03            # seeds to the plane threshold   0.03  大可能会导致过多的非接地点被错误地识别为接地点，从而导致地面分割不准确或者漏分割一些非地面点
th_dist: 0.03             # points to the plane threshold  0.03         如果你希望更少的点被认为是属于平面的点，你可以减小`th_dist`的值。这将要求点到平面的距离更接近于阈值，才会被归类为平面上的点。
                          # 变大这将允许更大的点到平面的距离被认为是属于平面的点。这可以有助于将更多的点归类为平面上的点，从而更好地分割出平面区域。
road_type: 1              # {1:八字, 2:直线, 3:track_drive}
vis: 0                    # 可视化选项（包括：euc 和 bbox）

# PassThrough settings
# general
z_up: 0.7                 # clip z up threshold (clip 截断)
z_down: -1.0              # clip z down threshold (default: -1.0)
# accelerate
accel_x_max: 30
accel_x_min: 0.5
accel_y_max: 2
accel_y_min: -2
# track
track_x_max: 20           # forward
track_x_min: 0.1          # backward
track_y_max: 5            # left
track_y_min: -5           # right

# skidpad (not in use)
skid_x_max: 30
skid_x_min: 0.1
skid_y_max: 70
skid_y_min: -70

# confidence settings
# 以下都为开区间
# skidpad mode ignores this

# cone height limits
min_height: 0.10
max_height: 0.35
# area size limits
min_area: 0.04
max_area: 0.2025
# bbox limits
max_box_altitude: 0.1

# model settings

sensor_model: 32          # velodyne product type

# vlp16
# str_range: "5,10,15,20"
# str_seg_distance: "0.18,0.35,0.53,0.6,0.6"

# vlp32
str_range: "5,10,15,20"         
str_seg_distance: "0.15,0.3,0.5,0.6,0.6"  # default value: 0.15,0.3,0.5,0.6,0.6
length:
  circle2lidar: 15.0

